concordium-contracts-common/src/schema.rs

```diff
@@ -1121,6 +1123,16 @@ mod impls {
             Ok(versioned_module_schema)
         }
 
+        /// Get a versioned module schema from a base64 string.
+        pub fn from_base64_str(s: &str) -> Result<Self, VersionedSchemaError> {
+            let bytes: Vec<u8> = general_purpose::STANDARD_NO_PAD
```

Comment:
In principle the input should always be padded, but using NO_PAD is the correct choice I think, since the padding is not really necessary, and this engine will accept both.